### PR TITLE
Nil hourly cost immunity

### PR DESCRIPTION
**Pull request by tbartynski**

### CODE REVIEW

File: spec/lib/atmosphere/utils_spec.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 require 'atmosphere/utils'
-#$: << '../../lib'
+# $: << '../../lib'
```
Comment (mkasztelnik) [inline spec/lib/atmosphere/utils_spec.rb]: Why this line is needed? Maybe we can remove it?

**Comment by mkasztelnik** [comment]
One small question and then :+1:

**Comment by mkasztelnik** [comment]
What is more are you able to squash these 6 commits into one?

File: spec/lib/atmosphere/utils_spec.rb (hunk 1 above)
Comment (tbartynski) [inline spec/lib/atmosphere/utils_spec.rb]: I didn't add this line. I only corrected it's style.

Comment (mkasztelnik) [inline spec/lib/atmosphere/utils_spec.rb]: I understand. If it is not needed then I propose to remove it.

Comment (tbartynski) [inline spec/lib/atmosphere/utils_spec.rb]: It was not needed. Removed it.

**Pull request merged**